integration/testutil.go

```diff
@@ -68,6 +68,8 @@ const (
 	dockerLibrary                = "public.ecr.aws/docker/library/"
 	blobStorePath                = "/var/lib/soci-snapshotter-grpc/content/blobs/sha256"
 	containerdBlobStorePath      = "/var/lib/containerd/io.containerd.content.v1.content/blobs/sha256"
+	OCI11RegistryImage           = "ghcr.io/oci-playground/registry:v3.0.0-alpha.1"
+	OCI10RegistryImage           = "docker.io/library/registry:2"
```

Comment:
nit: better to put them in the next const block with other `Image`, also mayber `OCI` -> `oci`

Reply:
+1 to non-exported.

I prefer to keep them out of the next image block because these are not used as SOCI test images, but rather as the base test infrastructure. I can add a comment to make that clear.

Reply:
SGTM